d2l-button-subtle.html

```diff
@@ -53,16 +53,13 @@
 			:host([h-align="text"]) button {
 				left: -0.6rem;
 			}
-			:host-context([dir="rtl"])[h-align="text"] button {
-				@apply --button-rtl;
-			}
```

Comment:
You'll need the this selector as well since shadycss doesn't properly convert multiple `:host` and `:host-context` selectors.

Reply:
I see - I removed this because it would result in the same behaviour (i.e., ignoring `host-context`)
It looks like the layout still works fine, because this rule
```css
:host([h-align="text"]) button {
    left: -0.6rem;
}
```
aligns it properly. Should this be left alone in that case?

Reply:
Yeah probably. I think this rule is needed for shadycss.